scripts/deploy.js

```diff
@@ -1,23 +1,70 @@
-const { deployStore } = require('./utils/deploy-store');
+const {
+  deployStore,
+  getCurrentAddressIfSameBytecode,
+} = require('./utils/deploy-store');
 const { getProxyAddress, proxyStore } = require('./utils/proxy-store');
 
-const CONTRACT_NAME = 'FleekERC721';
-const NETWORK = hre.network.name;
+// --- Inputs ---
 const ARGUMENTS = [
   'FleekNFAs', // Collection name
```

Comment:
I agree that we need to do it. I left that way just because it is faster to change it here during the development than writing in the CLI. For sure when we say this is a final version we would change this for CLI arguments.